From a7cc114fe0a6613c4c7ea0c3e1ebb8d927e671fc Mon Sep 17 00:00:00 2001
From: Miika Alikirri <miika.alikirri@suse.com>
Date: Fri, 4 Aug 2023 14:50:06 +0300
Subject: [PATCH] Use newer testing libraries for OBS

These newer testing libraries break the exsisting test
even though the test functionality is correct.
We cannot use the old libraries do to bug in node_module service
bug that prevents the obscpio file to have multiple versions
of the same library.

Note that this also requires the package-lock.json
in OBS to be manually updated.

---
 package.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package.json b/package.json
index 1ee15d5..2116544 100644
--- a/package.json
+++ b/package.json
@@ -22,8 +22,8 @@
     "@babel/preset-env": "^7.5.4",
     "@babel/preset-react": "^7.0.0",
     "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.2.2",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "14.4.3",
     "babel-eslint": "^10.0.3",
     "babel-loader": "^9.1.3",
     "chrome-remote-interface": "^0.33.0",
-- 
2.41.0

